modules/list.ts

```diff
@@ -20,6 +20,18 @@ import './optionsModal';
 import { itemManager } from './itemManager';
 import { exportDb } from './exportDb';
 
+// get the URL search param -> ?=LIST_ID
+const urlSearchParams = new URLSearchParams(window.location.search);
+const currentListId = urlSearchParams.get('id');
```

Comment:
This was a lot easier than explicitly passing the `listId` when we needed it - both on the /list page (list.ts in our bundle), and then subsequently in the `addItemForm` logic, the filtering logic, and so on and so on